src/Google.Api.Gax.Grpc/ServiceSettingsBase.cs

```diff
@@ -22,10 +22,10 @@ public abstract class ServiceSettingsBase
         protected ServiceSettingsBase()
         {
             UserAgent = new UserAgentBuilder()
+                .AppendDotNetEnvironment()
                 .AppendAssemblyVersion("gccl", GetType())
```

Comment:
Can you add this line:

```C#
AppendAssemblyVersion("gapic", GetType())
```

I know it's duplicative, but it will help keep things consistent across all languages.